app/models/my_academics/semesters.rb

```diff
@@ -153,22 +153,41 @@ def merge_semesters_count(data)
     end
 
     def map_enrollments(enrollment_term, term_id)
```

Comment:
What does this method do? Consider an intention revealing name

Reply:
@developish - that's a good call, but i did not introduce this method. I only modified it. I can change the name with low effort though so i will.

Reply:
@developish - that's a good call, but i did not introduce this method. I only modified it. I can change the name with low effort though so i will.

Reply:
`enrollment_term` should be plural, right?

Reply:
@developish - No. It's a single term (array), that has multiple enrollments.

Reply:
I know its been merged... but sounds like `term_enrollments` could have been a good name?